client-next/components/dashboard/link_tree_item.tsx

```diff
@@ -17,11 +19,19 @@ const LinkTreeItem = ({ link }: Props) => {
       description: "Olha que coisa mai linda",
     });
   }
+
+  const openModal = () => {
+    onEditClick(link);
+  };
+
+
+
   return (
     <div className="w-full relative">
       <div className="absolute top-1 right-2">
         <div onClick={cenas} className="clickable text-xs font-medium tracking-wide hover:text-primary">edit</div>
       </div>
+      <button onClick={openModal}>Open Modal</button>
```

Comment:
This should probably be named `Edit`?
```suggestion
      <button onClick={openModal}>Edit</button>
```